pkgs/applications/science/astronomy/stellarium/default.nix

```diff
@@ -37,6 +38,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-9uQ6u1+dSszmKG8eY6kSXhqsCPRGw6tulCTCrLByIxc=";
   };
 
+  # patch from fedora to allow building with Qt 6.10
+  patches = [
+    (fetchpatch2 {
```

Comment:
I think we should use fetchpatch here, to avoid unstable git hash length inside the patch?

Reply:
It is referencing an actual file and not a dynamically generated diff so it would be fine.